Sources/DolbyIORTSUIKit/Private/Managers/SettingsManager.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  SettingsManager.swift
+//
+
+import Combine
+import DolbyIORTSCore
+import Foundation
+
+public enum SettingsMode: Equatable {
+    case global
+    case stream(streamName: String, accountID: String)
+    
+    // Key against which `global settings` is stored in the `SettingsDictionary`
+    fileprivate static let keyGlobalStreamSettings = "global"
+
+    fileprivate var storageKey: String {
+        switch self {
+        case .global:
```

Comment:
Global and stream settings are App specific, I think we should not tie it to the UIKit. UIKit must expose flexible models to enable either settings

Reply:
I am not sure if stream settings is app specific, as it's very much coupled to the display logic of the StreamingScreen component itself. Also as the settings is backed by persistence, I am also curious to understand where should persistence live and whether it should propagated from App to UIKit or something internally managed within UIKit, because the settings mode today is only to used to distinguish which persistence data is being accessed.

Lets discuss in office tomorrow.

Reply:
Hi @RameshDPrasad - Lets discuss on the software design when we finish the Verizon changes. Merging the current PR for now as it has some refactors useful for the main branch.